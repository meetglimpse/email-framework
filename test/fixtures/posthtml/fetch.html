<extends src="test/stubs/layouts/basic.html">
  <block name="template">
    <fetch url="test/stubs/data.json">
      <each loop="user in response">
[[ user.name ]]
      </each>
    </fetch>
  </block>
</extends>
