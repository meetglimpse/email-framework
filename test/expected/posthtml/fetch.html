Leanne Graham

Ervin Howell

Clementine Bauch
